src/components/axes/axis.ts

```diff
@@ -30,15 +30,14 @@ module Plottable {
     private _rescaleCallback: ScaleCallback<Scale<D, number>>;
 
     /**
-     * Constructs an axis. An axis is a wrapper around a scale for rendering.
+     * Constructs an Axis.
+     * An Axis is a visual representation of a Scale.
```

Comment:
I'm not even sure if the description is that necessary. Are people going to use a charting library without knowing what an axis is?